apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -179,6 +179,12 @@ ConversationView.prototype = {
     return this.composerAccessors.addAttachment();
   },
 
+  takeAttachmentScreenshot: function(index) {
```

Comment:
> I'm sure @azasypkin has something like this in his head already ;)

Not really :) But I'd rather move this to ```composerAccessors``` and just reexpose here as we do for ```addAttachment```. Recently I was thinking about creating separate ```ComposeView``` that will use ```composeAccessors``` internally and we'll just need to call ```getComposeView``` on NewMessageView and ConversationView, so that we won't have to re-expose such methods for both NewMessage and Conversation - ComposeView will do it for us.